recipes/openjdk/all/conanfile.py

```diff
@@ -1,5 +1,6 @@
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools.files import copy, get, symlinks
+from conan.errors import ConanInvalidConfiguration
 import os
 
 required_conan_version = ">=1.33.0"
```

Comment:
I dont know what version symlink is but this should be close 🙈 

```suggestion
required_conan_version = ">=1.50.0"
```